ktor-network/jvm/src/io/ktor/network/selector/ActorSelectorManager.kt

```diff
@@ -32,34 +32,33 @@ public class ActorSelectorManager(context: CoroutineContext) : SelectorManagerSu
     @Volatile
     private var closed = false
 
-    private val mb = LockFreeMPSCQueue<Selectable>()
+    private val messages = LockFreeMPSCQueue<Selectable>()
 
     override val coroutineContext: CoroutineContext = context + CoroutineName("selector")
 
     init {
         launch {
-            val selector = provider.openSelector() ?: error("openSelector() = null")
+            val selector = provider.openSelector() ?: error("Failed to open selector.")
             selectorRef = selector
-            try {
+
+            selector.use {
                 try {
-                    process(mb, selector)
+                    process(messages, it)
```

Comment:
I would prefer to see `selector` instead of `it` here since the `use` block is quite big and it's not obvious, what `it` represents

Reply:
Sure